RNSound/RNSound.m

```diff
@@ -196,6 +197,7 @@ -(NSDictionary *)constantsToExport {
   if (player) {
     [[self callbackPool] setObject:[callback copy] forKey:key];
     [player play];
+    [self setOnPlay:YES forKey:key];
```

Comment:
How about renaming this `key` to `playerKey`? I'm suggesting this because I can't tell what's the exact meaning of this `key` in the native side. At a first glance, I was confused between `Is it event key?` or `Is it player key?`.
  
The above comment might affect the `nextKey` global variable in `sound.js`. Every variable should tell the purpose of itself inline without referring other scopes.

Reply:
Yes! I do agree with you. But the variable name `key` and `nextKey` is given by the upstream library, so I'm afraid that might cause too much differences from the upstream library. I still can change `forKey` to `forPlayerKey` in the above case though!